seqr/views/apis/individual_api_tests.py

```diff
@@ -463,6 +513,310 @@ def test_individuals_table_handler(self):
         response = self.client.post(save_url)
         self.assertEqual(response.status_code, 200)
 
+    @mock.patch('seqr.views.utils.permissions_utils.PM_USER_GROUP', 'project-managers')
+    @mock.patch('seqr.views.utils.pedigree_info_utils.NO_VALIDATE_MANIFEST_PROJECT_CATEGORIES')
+    @mock.patch('seqr.utils.communication_utils.EmailMultiAlternatives')
+    def test_individuals_sample_manifest_table_handler(self, mock_email, mock_no_validate_categories):
+        receive_url = reverse(receive_individuals_table_handler, args=[PROJECT_GUID])
+        self.check_manager_login(receive_url)
+
+        def _send_request_data(data):
+            return self.client.post(receive_url, {'f': SimpleUploadedFile(
+                'sample_manifest.tsv', '\n'.join(['\t'.join(row) for row in data]).encode('utf-8')),
+            })
+
+        header_2 = [
+            'Kit ID', 'Well', 'Sample ID', 'Family ID', 'Alias', 'Alias', 'Paternal Sample ID', 'Maternal Sample ID',
+            'Gender', 'Affected Status', 'Primary Biosample', 'Analyte Type', 'Tissue Affected Status', 'Recontactable',
+            'Volume', 'Concentration', 'Notes', 'MONDO Label', 'MONDO ID', 'Consent Code', 'Data Use Restrictions']
+        header_3 = [
+            '', 'Position', '', '', 'Collaborator Participant ID', 'Collaborator Sample ID', '', '', '', '', '', '',
+            '(i.e yes, no)', '(i.e yes, no, unknown)', 'ul', 'ng/ul', '', '', '(i.e. "MONDO:0031632")', '',
+            'indicate study/protocol number']
+        data = [
+            ['Do not modify - Broad use', '', '', 'Please fill in columns D - T', '', '', '', '', '', '', '', '', '',
+             '', '', '', '', '', '', '', ''],
+            header_2, header_3,
+        ]
+
+        response = _send_request_data(data)
+        self.assertEqual(response.status_code, 400)
+        self.assertDictEqual(response.json(), {
+            'errors': ['Error while parsing file: sample_manifest.tsv. Unsupported file format'], 'warnings': [],
+        })
+
+        data[1] = header_2[:5] + header_2[7:10] + header_2[14:17] + ['Coded Phenotype'] + header_2[19:]
+        response = _send_request_data(data)
+        self.assertDictEqual(response.json(), {
+            'errors': ['Error while parsing file: sample_manifest.tsv. Unsupported file format'], 'warnings': [],
```

Comment:
You can add comments about the two tests' different purposes. Such as `test incomplete headers`.